source/MRViewer/MRViewport.h

```diff
@@ -344,22 +344,18 @@ class Viewport
     {
         Visible, // fit all visible objects
         SelectedPrimitives, // fit only selected primitives
-        SelectedObjects // fit only selected objects
+        SelectedObjects, // fit only selected objects
+        CustomObjectsList // fit only given objects (need additional objects list)
     };
     struct FitDataParams
     {
-        float factor{ 1.f }; // part of the screen for scene location
+        float factor{ 0.9f }; // part of the screen for scene location
         // snapView - to snap camera angle to closest canonical quaternion
         // orthographic view: camera moves a bit, fit FOV by the whole width or height
         // perspective view: camera is static, fit FOV to closest border.
         bool snapView{ false };
         FitMode mode{ FitMode::Visible }; // fit mode
-
-        FitDataParams( float factor_ = 1.f, bool snapView_ = false, FitMode mode_ = FitMode::Visible ) :
-            factor( factor_ ),
-            snapView( snapView_ ),
-            mode( mode_ )
-        {};
+        std::vector<std::shared_ptr<VisualObject>> objsList;
```

Comment:
please add comment that it is used only in CustomObjectsList mode

Reply:
fixed